apis/composition.yaml

```diff
@@ -0,0 +1,22 @@
+apiVersion: apiextensions.crossplane.io/v1
+kind: Composition
+metadata:
+  name: xnopresources.nop.example.org
+spec:
+  compositeTypeRef:
+    apiVersion: nop.example.org/v1alpha1
+    kind: XNopResource
+  resources:
```

Comment:
should we default to using a Functions based approach instead? Might be nice to steer folks that direction now